src/app/store/slices/workspaces/workspacesStore.ts

```diff
@@ -324,6 +327,17 @@ export const workspacesSlice = createSlice({
 
         state.isLoadingWorkspaces = false;
         notificationsService.show({ text: 'checking workspaces error ' + errorMsg, type: ToastType.Warning });
+      })
+
+      .addCase(editWorkspace.rejected, () => {
+        notificationsService.show({ text: t('error.serverError'), type: ToastType.Warning });
```

Comment:
Show more user friendly message instead of server error.
Something like "There was an error updating your details. Please try again in a few minutes"